src/test/java/edu/kis/powp/jobs2d/VisitorTest.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.powp.jobs2d;
+
+import edu.kis.powp.jobs2d.command.*;
+import edu.kis.powp.jobs2d.command.gui.CommandManagerWindow;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class VisitorTest implements ActionListener {
+	private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+	@Override
+	public void actionPerformed(ActionEvent e) {
+		List<DriverCommand> commands = new ArrayList<>();
+
+		commands.add(new SetPositionCommand(-50, -50));
+		commands.add(new OperateToCommand(-50, 50));
+		commands.add(new OperateToCommand(50, 50));
+		commands.add(new OperateToCommand(50, -50));
+		commands.add(new OperateToCommand(-50, -50));
```

Comment:
Please use the command loaded in the CommandManager (getCurrentCommand).

If you want to provide your own command for testing please use CompoundCommand implementation from Pull Request #4 and provide a simple factory to generate the command, and user test option to load it to the CommandManager.